### PR TITLE
Added news & quantstart commands to the telegram bot

### PR DESCRIPTION
**Pull request by shivanshsinghal107**
### PR for [#6 ](https://github.com/siddeshshewde/Projects/issues/6)
I have added a total of 3 commands for the bot:
- `/news` - fetch latest finance news articles from CNBC
- `/quantstart` - fetch latest Algo trading articles from Quantstart
- `/help` - display all bot commands

Try running these in telegram & let me know if they're working well.

### CODE REVIEW

**Comment by shivanshsinghal107** [comment]
I just wanted to clarify that PRs for hacktoberfest count only if either The PR is labelled as hacktoberfest-accepted by a maintainer OR Submitted in a repo with the hacktoberfest topic.
So just to make sure this contribution count towards hacktoberfest can you do either of the two things.

Thanks

**Comment by siddeshshewde** [comment]
that was pretty fast. Will check and get back

**Comment by shivanshsinghal107** [comment]
> that was pretty fast. Will check and get back

Yeah because I have worked on telegram bot as well as a lot web scraping before.

**Comment by shivanshsinghal107** [comment]
Thank you so much

**Comment by shivanshsinghal107** [comment]
Hey, I didn't hear from you, are the commands working well?

**Comment by siddeshshewde** [comment]
I just checked the code as I have limited access to my laptop for now. Will test it by tonight.

Just wanted to check if we don't need to add 'bot' as an argument/parameter in all the functions which you have defined?
as you are using context.bot.send_message inside the function.

**Comment by shivanshsinghal107** [comment]
Ok. To answer your question this `context` here is an object which stores information regarding any update, error, or job. And bot can be accessed from context according to the latest version of the telegram library.
You can have more info regarding context callbacks here:
https://github.com/python-telegram-bot/python-telegram-bot/wiki/Transition-guide-to-Version-12.0#what-exactly-is-callbackcontext

**Comment by siddeshshewde** [comment]
Trying it out now. 

I tried running: pip install python-telegram-bot --upgrade
and got this output:

Installing collected packages: python-telegram-bot
Successfully installed python-telegram-bot-12.8

Here version seems to be 12.8.
Any idea how can I upgrade to v13?

**Comment by siddeshshewde** [comment]
> Ok. To answer your question this `context` here is an object which stores information regarding any update, error, or job. And bot can be accessed from context according to the latest version of the telegram library.
> You can have more info regarding context callbacks here:
> https://github.com/python-telegram-bot/python-telegram-bot/wiki/Transition-guide-to-Version-12.0#what-exactly-is-callbackcontext

I went through the docs, thank you for the explanation (y)

**Comment by shivanshsinghal107** [comment]
> Trying it out now.
> 
> I tried running: pip install python-telegram-bot --upgrade
> and got this output:
> 
> Installing collected packages: python-telegram-bot
> Successfully installed python-telegram-bot-12.8
> 
> Here version seems to be 12.8.
> Any idea how can I upgrade to v13? Looks like 12.8 is the latest version: https://pypi.org/project/python-telegram-bot/

Yeah actually, pypi shows the latest versions of libraries, so 12.8 is the latest

**Comment by siddeshshewde** [comment]
> > Trying it out now.
> > I tried running: pip install python-telegram-bot --upgrade
> > and got this output:
> > Installing collected packages: python-telegram-bot
> > Successfully installed python-telegram-bot-12.8
> > Here version seems to be 12.8.
> > Any idea how can I upgrade to v13? Looks like 12.8 is the latest version: https://pypi.org/project/python-telegram-bot/
> 
> Yeah actually, pypi shows the latest versions of libraries, so 12.8 is the latest

Got this error:
  File ".\telegram_bot.py", line 54, in start
    context.bot.send_message(chat_id = update.effective_chat.id, text = '''
AttributeError: 'Update' object has no attribute 'bot'

Do you think you could comment out the v13 code for now and make it compatible for v12.8?
We can update the code later on for v13

File: Telegram Bot (Python)/telegram_bot.py [hunk 1]
```diff
@@ -1,40 +1,90 @@
...
+
+# function to get list of all commands
+def get_commands(update, context):
+    context.bot.send_message(chat_id = update.effective_chat.id, text = '''
+    /news - get the latest finance news articles
+    /quantstart - get top articles of Algorithmic Trading from Quantstart
+    /help - get list of commands of the bot''')
 
 def main():
     updater = Updater('1115939597:AAFJr-WpQz96yJ_ulJXwNApv_q32WuMg2Jk')
```
Comment (shivanshsinghal107) [inline Telegram Bot (Python)/telegram_bot.py:80]: I think I got the problem, here in v12 we have to pass an argument `use_context = True` in the updater. So:
```python
updater = Updater('1115939597:AAFJr-WpQz96yJ_ulJXwNApv_q32WuMg2Jk', use_context = True)
```
Reference:
https://github.com/python-telegram-bot/python-telegram-bot/wiki/Transition-guide-to-Version-12.0#context-based-callbacks

**Comment by siddeshshewde** [comment]
Nice work bro

**Pull request merged**